packages/amplify_auth_cognito/example/.gitignore

```diff
@@ -43,7 +43,7 @@ app.*.map.json
 # Exceptions to above rules.
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 
-#amplify
+#amplify-do-not-edit-begin
```

Comment:
Is this what the current version of the CLI is outputting now?

Reply:
Yep